aquacomputer_d5next.c

```diff
@@ -1016,6 +1046,9 @@ static int aqc_write(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 {
 	int ret, pwm_value, temp_sensor;
 	long ctrl_mode;
+	int offset[3];
```

Comment:
Name these `ctrl_values_offsets`, `ctrl_values`, `ctrl_values_sizes` or similar, so it's clear what they are for. And add a comment above that they're for setting multiple values at once in the ctrl report.